docs/tutorial/bitwise.md

```diff
@@ -114,11 +114,11 @@ module {
 
 This implementation uses the fact that we can combine two values into a single value and apply a single table lookup to this combined value!
 
-There are two major problems with this implementation though:
+There are two major problems with this implementation:
 1) packing requires the same bit-width across operands.
 2) packing requires the bit-width of at least `x.bit_width + y.bit_width` and that bit-width cannot exceed maximum TLU bit-width, which is `16` at the moment.
 
-What this means is if we are comparing `uint3` and `uint6`, we need to convert both of them to `uint9` in some way to do the packing and proceed with the TLU in 9-bits. There are 4 ways to achieve this behavior.
+This means that if we are comparing `uint3` and `uint6`, we need to convert both of them to `uint9`. After this we are able to do the packing and can proceed with the TLU in 9-bits. There are 4 ways to achieve this.
```

Comment:
It was specifically written `in some way` because there are exactly two ways to do this (casted or promoted), which are explained below.

Reply:
> There are 4 ways to achieve this

casted or promoted for lhs = 2 possibilities
casted or promoted for rhs = 2 possibilities
in total = 4 possibilities